src/sentry/static/sentry/app/views/organizationIntegrations/integrationDirectoryAppInstallationDetail.tsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import IntegrationDirectoryApplicationRow from 'app/views/settings/organizationDeveloperSettings/sentryApplicationRow/integrationDirectoryApplicationRow';
+import SentryTypes from 'app/sentryTypes';
+import {Organization, SentryApp, SentryAppInstallation} from 'app/types';
+
+type Props = {
+  organization: Organization;
+  install?: SentryAppInstallation;
+  app: SentryApp;
+};
+
+class SentryAppInstallationDetail extends React.Component<Props> {
+  static propTypes = {
```

Comment:
@NisanthanNanthakumar Let's not add `propTypes` to any new components. The TS props are enough.